ModelBundle/DataFixtures/MongoDB/DemoContent/NodeRootDataGenerator.php

```diff
@@ -0,0 +1,188 @@
+<?php
+
+namespace OpenOrchestra\ModelBundle\DataFixtures\MongoDB\DemoContent;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use OpenOrchestra\ModelBundle\Document\Block;
+use OpenOrchestra\ModelBundle\Document\Node;
+use OpenOrchestra\DisplayBundle\DisplayBlock\Strategies\TinyMCEWysiwygStrategy;
+use OpenOrchestra\ModelInterface\Model\NodeInterface;
+use OpenOrchestra\ModelBundle\Document\Area;
+
+/**
+ * Class NodeRootDataGenerator
+ */
+class NodeRootDataGenerator extends AbstractDataGenerator
```

Comment:
this generator is used only in functional fixture so can you rename it to  NodeRootFunctionalDataGenerator